pretyping/recordops.ml

```diff
@@ -243,7 +243,7 @@ let compute_canonical_projections env ~warn (gref,ind) =
 
 let pr_cs_pattern = function
     Const_cs c -> Nametab.pr_global_env Id.Set.empty c
-  | Prod_cs -> str "_ -> _"
+  | Prod_cs -> str "forall _, _"
```

Comment:
Just to play safe... @pi8027 does your tool for detecting errors in packed classes parse "_ -> _" ?
I'm fine with the renaming, I'm just giving an heads up to Kazuhiko that has a tool parsing the output of Print Canonical Projections.

Reply:
No. If an input line does not match with the regexp (roughly `.*\.sort <- .*\.sort ( .* )`), my tool ignores it. https://github.com/math-comp/math-comp/blob/master/etc/utils/hierarchy.ml#L59-L73